src/api/fuel.js

```diff
@@ -0,0 +1,531 @@
+/* eslint-disable quotes */
+// Most of this code was taken and addapted from https://gist.github.com/aaroncox/d74a73b3d9fbc20836c32ea9deda5d70
+import {
+  User
+} from 'universal-authenticator-library';
+import {
+  APIClient,
+  Name,
+  PackedTransaction,
+  PermissionLevel,
+  Serializer,
+  Transaction
+} from '@greymass/eosio';
+import { Dialog } from 'quasar';
+
+// this simulates the getChain function from OBE
+const getChain = () => ({
+  getHyperionEndpoint: () => (process.env.HYPERION_ENDPOINT),
+  getFuelRPCEndpoint: () => { return process.env.fuelrpc; },
+});
+
+// The maximum fee per transaction this script is willing to accept
+const maxFee = 0.05;
+
+// expire time in millisec
+const expireSeconds = 3600;
+
+const chain = getChain();
+const client = new APIClient({
+  url: chain.getHyperionEndpoint()
+});
+
+const fuelrpc = chain.getFuelRPCEndpoint();
+const resourceProviderEndpoint = `${fuelrpc?.protocol}://${fuelrpc?.host}:${fuelrpc?.port}/v1/resource_provider/request_transaction`;
+
+
+// Wrapper for the user to intersect the signTransaction call
+export class FuelUserWrapper extends User {
+  user = null;
+  constructor(user/*: User*/) {
+    super();
+    this.user = user;
+  }
+
+  async signTransaction(
+    originalTransaction/*: AnyTransaction*/,
+    originalconfig/*: SignTransactionConfig*/
+  )/*: Promise<SignTransactionResponse>*/ {
```

Comment:
why are these commented out? 🤔

Reply:
Yes, the idea es to enable all types once ty is enabled in this project.
@karynemayer, do you prefer them to be removed?